### ISSUE TITLE
Connection Refused error ( Intermittent )

### ISSUE THREAD

**Issue by Exillium313** [WooMinecraft/WooMinecraft#208]
So my check interval is to default 15 min, however randomly the checks result in this error.  Never had this issue until recently, cannot recall anything ive changed that might cause it though.  Sometimes the checks go through just fine, other times they fail and result with this error.

Any help is appreciated.

Error Log: 
https://pastebin.com/7V1ZjAuQ

### Server Information:
* **Host:**siteground.com
* **URL:**pandoriamc.com

### Config.yml
* **url:** https://pastebin.com/MMpMedLK

**Comment by ethsmith** [comment]
Means Java can't connect to the server but it happens randomly... packet loss maybe?

**Comment by Exillium313** [comment]
I must've overlooked it before, but I just read about making sure there are no redirects.  I'm currently using my domain to redirect to my minecraft server IP for connecting.  

Would my a DNS, A Record pointing from my url pandoriamc.com to my servers IP cause this issue?

**Comment by ethsmith** [comment]
Like pandoriamc.com will go to the website and the server? I could see how that would cause issues but I haven't actually ever tested that case before. I always had a store subdomain like store.website.com

**Comment by Exillium313** [comment]
Yes it does both, due to DNS limitations of siteground I couldn't make an SRV record to change the port of minecraft and make play.pandoriamc.com. I've since switched ports to minecraft default which has resolved that issue and a simple A Record pointing play.pandoriamc.com to my servers IP works to connect to my server
